tools/wpt/browser.py

```diff
@@ -633,7 +633,9 @@ def find_nightly_binary(self, dest):
 
     def find_binary(self, venv_path=None, channel=None):
         if channel == "nightly":
-            return self.find_nightly_binary(self._get_dest(venv_path, channel))
+            nightly_binary = self.find_nightly_binary(self._get_dest(venv_path, channel))
+            if nightly_binary is not None:
```

Comment:
So the upshot of this is that if we try to find a nightly binary and fail, we'll fall back to using just `google-chrome` on the `$PATH` on linux and similar default names on other channels? It's unclear to me if that's expected.

Reply:
That is the intent, the current implementation will just throw a generic error if a nightly binary doesn't exist, no falling back. But maybe change is not a good idea from some usability perspective and falling back could be an issue?

Reply:
It looks like Firefox has a similar implementation where the virtual environment is checked, and then falls back if a binary is not located there:

https://github.com/web-platform-tests/wpt/blob/ab2592d3c62f8f131d695e7c5c0ab08e273c85ce/tools/wpt/browser.py#L268-L285

Reply:
The difference is that Firefox doesn't have different binary names for nightly vs stable. Since Chrome does, I'm not sure if it makes sense to use something that's known to not be a nightly build when the channel was specified as nightly.

Reply:
I see, thank you for the clarification. I am bringing this up because this was a pain point when I first tried to run the tests in this repository. Do you think it would be better to simply add a more verbose error message for this section? "nightly" is also the default choice when you do not specify a channel and run the tests, so this issue comes up whenever a user specifies nothing.

https://github.com/web-platform-tests/wpt/blob/ae28863b68291fa71320ccc577aa874fbb7009b7/tools/wpt/install.py#L27-L37

Reply:
I updated the code to no longer fall back to looking on the `$PATH` or other channels if a browser binary is not found on the virtualenv path. This should behave exactly as the current implementation except a more verbose error message will display on how to fix the issue.

Reply:
@jgraham The more I thought about this, the more it feels like a separate issue from the main change. The most important part of this PR is the `venv_path` change, which fixes an issue where the Chrome browser binary would go completely undetected even if it existed in virtualenv. I removed the above change, and if it's still important, I will return to it when I am certain of the correct approach.